Storybook: cannot serve E2e storybook

### Description

I'm not sure if this is an issue with my environment, so I would appreciate it if you could test it.

<details><summary>Log</summary>

```
[0] [1] 1:34:02 PM - Starting compilation in watch mode...
[0] [1] 
[1] WARN The following packages are incompatible with Storybook 8.4.7 as they depend on different major versions of Storybook packages:
[1] WARN
[1]  - storybook-source-link@2.0.9
[1] WARN  Repo: https://storybook.js.org/addons/storybook-source-link/
[1] WARN 
[1] WARN 
[1] WARN Please consider updating your packages or contacting the maintainers for compatibility details.
[1] WARN For more on Storybook 8 compatibility, see the linked GitHub issue:
[1] WARN https://github.com/storybookjs/storybook/issues/26031
[1] info => Starting manager..
[1] info => Starting preview..
[1] info => Using implicit CSS loaders
[1] info => Using default Webpack5 setup
[1] <i> [webpack-dev-middleware] wait until bundle finished
[1] <s> [webpack.Progress] 3% setup watch run
[1] <s> [webpack.Progress] 3% setup watch run VirtualModulesPlugin
[1] <s> [webpack.Progress] 3% setup watch run webpack-dev-middleware
[1] <s> [webpack.Progress] 3% setup watch run
[1] <s> [webpack.Progress] 4% setup normal module factory
[1] <s> [webpack.Progress] 4% setup normal module factory CaseSensitivePathsPlugin
[1] <s> [webpack.Progress] 4% setup normal module factory
[1] <s> [webpack.Progress] 5% setup context module factory
[1] <s> [webpack.Progress] 5% setup context module factory
[1] <s> [webpack.Progress] 6% setup before compile
[1] <s> [webpack.Progress] 6% setup before compile ProgressPlugin
[1] <s> [webpack.Progress] 6% setup before compile ProgressPlugin
[1] <s> [webpack.Progress] 6% setup before compile
[1] <s> [webpack.Progress] 7% setup compile
[1] <s> [webpack.Progress] 7% setup compile ExternalsPlugin
[1] <s> [webpack.Progress] 7% setup compile ExternalsPlugin
[1] <s> [webpack.Progress] 7% setup compile
[1] <s> [webpack.Progress] 8% setup compilation
[1] <s> [webpack.Progress] 8% setup compilation HtmlWebpackPlugin
[1] <s> [webpack.Progress] 8% setup compilation ArrayPushCallbackChunkFormatPlugin
[1] <s> [webpack.Progress] 8% setup compilation JsonpChunkLoadingPlugin
[1] <s> [webpack.Progress] 8% setup compilation StartupChunkDependenciesPlugin
[1] <s> [webpack.Progress] 8% setup compilation ImportScriptsChunkLoadingPlugin
[1] <s> [webpack.Progress] 8% setup compilation WorkerPlugin
[1] <s> [webpack.Progress] 8% setup compilation SplitChunksPlugin
[1] <s> [webpack.Progress] 8% setup compilation RuntimeChunkPlugin
[1] <s> [webpack.Progress] 8% setup compilation ResolverCachePlugin
[1] <s> [webpack.Progress] 8% setup compilation
[1] <s> [webpack.Progress] 9% setup compilation
[1] <s> [webpack.Progress] 9% setup compilation DefinePlugin
[1] <s> [webpack.Progress] 9% setup compilation ProvidePlugin
[1] <s> [webpack.Progress] 9% setup compilation HotModuleReplacementPlugin
[1] <s> [webpack.Progress] 9% setup compilation ProgressPlugin
[1] <s> [webpack.Progress] 9% setup compilation DefinePlugin
[1] <s> [webpack.Progress] 9% setup compilation ChunkPrefetchPreloadPlugin
[1] <s> [webpack.Progress] 9% setup compilation ModuleInfoHeaderPlugin
[1] <s> [webpack.Progress] 9% setup compilation SourceMapDevToolPlugin
[1] <s> [webpack.Progress] 9% setup compilation JavascriptModulesPlugin
[1] <s> [webpack.Progress] 9% setup compilation JsonModulesPlugin
[1] <s> [webpack.Progress] 9% setup compilation AssetModulesPlugin
[1] <s> [webpack.Progress] 9% setup compilation EntryPlugin
[1] <s> [webpack.Progress] 9% setup compilation EntryPlugin
[1] <s> [webpack.Progress] 9% setup compilation RuntimePlugin
[1] <s> [webpack.Progress] 9% setup compilation InferAsyncModulesPlugin
[1] <s> [webpack.Progress] 9% setup compilation DataUriPlugin
[1] <s> [webpack.Progress] 9% setup compilation FileUriPlugin
[1] <s> [webpack.Progress] 9% setup compilation CompatibilityPlugin
[1] <s> [webpack.Progress] 9% setup compilation HarmonyModulesPlugin
[1] <s> [webpack.Progress] 9% setup compilation AMDPlugin
[1] <s> [webpack.Progress] 9% setup compilation RequireJsStuffPlugin
[1] <s> [webpack.Progress] 9% setup compilation CommonJsPlugin
[1] <s> [webpack.Progress] 9% setup compilation LoaderPlugin
[1] <s> [webpack.Progress] 9% setup compilation LoaderPlugin
[1] <s> [webpack.Progress] 9% setup compilation NodeStuffPlugin
[1] <s> [webpack.Progress] 9% setup compilation APIPlugin
[1] <s> [webpack.Progress] 9% setup compilation ExportsInfoApiPlugin
[1] <s> [webpack.Progress] 9% setup compilation WebpackIsIncludedPlugin
[1] <s> [webpack.Progress] 9% setup compilation ConstPlugin
[1] <s> [webpack.Progress] 9% setup compilation UseStrictPlugin
[1] <s> [webpack.Progress] 9% setup compilation RequireIncludePlugin
[1] <s> [webpack.Progress] 9% setup compilation RequireEnsurePlugin
[1] <s> [webpack.Progress] 9% setup compilation RequireContextPlugin
[1] <s> [webpack.Progress] 9% setup compilation ImportPlugin
[1] <s> [webpack.Progress] 9% setup compilation ImportMetaContextPlugin
[1] <s> [webpack.Progress] 9% setup compilation SystemPlugin
[1] <s> [webpack.Progress] 9% setup compilation ImportMetaPlugin
[1] <s> [webpack.Progress] 9% setup compilation URLPlugin
[1] <s> [webpack.Progress] 9% setup compilation DefaultStatsFactoryPlugin
[1] <s> [webpack.Progress] 9% setup compilation DefaultStatsPresetPlugin
[1] <s> [webpack.Progress] 9% setup compilation DefaultStatsPrinterPlugin
[1] <s> [webpack.Progress] 9% setup compilation JavascriptMetaInfoPlugin
[1] <s> [webpack.Progress] 9% setup compilation EnsureChunkConditionsPlugin
[1] <s> [webpack.Progress] 9% setup compilation RemoveEmptyChunksPlugin
[1] <s> [webpack.Progress] 9% setup compilation MergeDuplicateChunksPlugin
[1] <s> [webpack.Progress] 9% setup compilation SideEffectsFlagPlugin
[1] <s> [webpack.Progress] 9% setup compilation FlagDependencyExportsPlugin
[1] <s> [webpack.Progress] 9% setup compilation NamedModuleIdsPlugin
[1] <s> [webpack.Progress] 9% setup compilation NamedChunkIdsPlugin
[1] <s> [webpack.Progress] 9% setup compilation DefinePlugin
[1] <s> [webpack.Progress] 9% setup compilation TemplatedPathPlugin
[1] <s> [webpack.Progress] 9% setup compilation RecordIdsPlugin
[1] <s> [webpack.Progress] 9% setup compilation WarnCaseSensitiveModulesPlugin
[1] <s> [webpack.Progress] 9% setup compilation IgnoreWarningsPlugin
[1] <s> [webpack.Progress] 9% setup compilation ProgressPlugin
[1] <s> [webpack.Progress] 9% setup compilation
[1] <s> [webpack.Progress] 10% building
[1] <s> [webpack.Progress] 10% building 0/1 entries 0/0 dependencies 0/0 modules
[1] <s> [webpack.Progress] 37% building 1/2 entries 39/40 dependencies 18/27 modules
[1] <s> [webpack.Progress] 37% building import loader ./storybook/webpack/source-link-loader.js
[1] <s> [webpack.Progress] 37% building import loader ./storybook/webpack/copy-tags-to-badges.js
[1] <s> [webpack.Progress] 37% building 1/2 entries 58/59 dependencies 20/36 modules
[1] <s> [webpack.Progress] 65% building 2/2 entries 83/83 dependencies 51/51 modules
[1] <s> [webpack.Progress] 65% building
[1] <s> [webpack.Progress] 69% building finish
[1] <s> [webpack.Progress] 69% building finish
[1] <s> [webpack.Progress] 70% sealing finish module graph
[1] <s> [webpack.Progress] 70% sealing finish module graph ResolverCachePlugin
[1] <s> [webpack.Progress] 70% sealing finish module graph InferAsyncModulesPlugin
[1] <s> [webpack.Progress] 70% sealing finish module graph FlagDependencyExportsPlugin
[1] <s> [webpack.Progress] 70% sealing finish module graph
[1] <s> [webpack.Progress] 70% sealing plugins
[1] <s> [webpack.Progress] 70% sealing plugins WarnCaseSensitiveModulesPlugin
[1] <s> [webpack.Progress] 70% sealing plugins
[1] <s> [webpack.Progress] 71% sealing dependencies optimization
[1] <s> [webpack.Progress] 71% sealing dependencies optimization SideEffectsFlagPlugin
[1] <s> [webpack.Progress] 71% sealing dependencies optimization
[1] <s> [webpack.Progress] 71% sealing after dependencies optimization
[1] <s> [webpack.Progress] 71% sealing after dependencies optimization
[1] <s> [webpack.Progress] 72% sealing chunk graph
[1] <s> [webpack.Progress] 72% sealing chunk graph
[1] <s> [webpack.Progress] 73% sealing after chunk graph
[1] <s> [webpack.Progress] 73% sealing after chunk graph
[1] <s> [webpack.Progress] 73% sealing optimizing
[1] <s> [webpack.Progress] 73% sealing optimizing
[1] <s> [webpack.Progress] 74% sealing module optimization
[1] <s> [webpack.Progress] 74% sealing module optimization
[1] <s> [webpack.Progress] 75% sealing after module optimization
[1] <s> [webpack.Progress] 75% sealing after module optimization
[1] <s> [webpack.Progress] 75% sealing chunk optimization
[1] <s> [webpack.Progress] 75% sealing chunk optimization EnsureChunkConditionsPlugin
[1] <s> [webpack.Progress] 75% sealing chunk optimization RemoveEmptyChunksPlugin
[1] <s> [webpack.Progress] 75% sealing chunk optimization MergeDuplicateChunksPlugin
[1] <s> [webpack.Progress] 75% sealing chunk optimization SplitChunksPlugin
[1] <s> [webpack.Progress] 75% sealing chunk optimization RemoveEmptyChunksPlugin
[1] <s> [webpack.Progress] 75% sealing chunk optimization
[1] <s> [webpack.Progress] 76% sealing after chunk optimization
[1] <s> [webpack.Progress] 76% sealing after chunk optimization
[1] <s> [webpack.Progress] 77% sealing module and chunk tree optimization
[1] <s> [webpack.Progress] 77% sealing module and chunk tree optimization PersistentChildCompilerSingletonPlugin
[1] <s> [webpack.Progress] 77% sealing module and chunk tree optimization
[1] <s> [webpack.Progress] 77% sealing after module and chunk tree optimization
[1] <s> [webpack.Progress] 77% sealing after module and chunk tree optimization
[1] <s> [webpack.Progress] 78% sealing chunk modules optimization
[1] <s> [webpack.Progress] 78% sealing chunk modules optimization
[1] <s> [webpack.Progress] 78% sealing after chunk modules optimization
[1] <s> [webpack.Progress] 78% sealing after chunk modules optimization
[1] <s> [webpack.Progress] 79% sealing module reviving
[1] <s> [webpack.Progress] 79% sealing module reviving RecordIdsPlugin
[1] <s> [webpack.Progress] 79% sealing module reviving
[1] <s> [webpack.Progress] 80% sealing before module ids
[1] <s> [webpack.Progress] 80% sealing before module ids
[1] <s> [webpack.Progress] 80% sealing module ids
[1] <s> [webpack.Progress] 80% sealing module ids NamedModuleIdsPlugin
[1] <s> [webpack.Progress] 80% sealing module ids
[1] <s> [webpack.Progress] 81% sealing module id optimization
[1] <s> [webpack.Progress] 81% sealing module id optimization
[1] <s> [webpack.Progress] 82% sealing module id optimization
[1] <s> [webpack.Progress] 82% sealing module id optimization
[1] <s> [webpack.Progress] 82% sealing chunk reviving
[1] <s> [webpack.Progress] 82% sealing chunk reviving RecordIdsPlugin
[1] <s> [webpack.Progress] 82% sealing chunk reviving
[1] <s> [webpack.Progress] 83% sealing before chunk ids
[1] <s> [webpack.Progress] 83% sealing before chunk ids
[1] <s> [webpack.Progress] 84% sealing chunk ids
[1] <s> [webpack.Progress] 84% sealing chunk ids NamedChunkIdsPlugin
[1] <s> [webpack.Progress] 84% sealing chunk ids
[1] <s> [webpack.Progress] 84% sealing chunk id optimization
[1] <s> [webpack.Progress] 84% sealing chunk id optimization
[1] <s> [webpack.Progress] 85% sealing after chunk id optimization
[1] <s> [webpack.Progress] 85% sealing after chunk id optimization
[1] <s> [webpack.Progress] 86% sealing record modules
[1] <s> [webpack.Progress] 86% sealing record modules RecordIdsPlugin
[1] <s> [webpack.Progress] 86% sealing record modules
[1] <s> [webpack.Progress] 86% sealing record chunks
[1] <s> [webpack.Progress] 86% sealing record chunks RecordIdsPlugin
[1] <s> [webpack.Progress] 86% sealing record chunks
[1] <s> [webpack.Progress] 87% sealing module hashing
[1] <s> [webpack.Progress] 87% sealing module hashing
[1] <s> [webpack.Progress] 87% sealing code generation
[1] <s> [webpack.Progress] 87% sealing code generation
[1] <s> [webpack.Progress] 88% sealing runtime requirements
[1] <s> [webpack.Progress] 88% sealing runtime requirements
[1] <s> [webpack.Progress] 89% sealing hashing
[1] <s> [webpack.Progress] 89% sealing hashing
[1] <s> [webpack.Progress] 89% sealing after hashing
[1] <s> [webpack.Progress] 89% sealing after hashing
[1] <s> [webpack.Progress] 90% sealing record hash
[1] <s> [webpack.Progress] 90% sealing record hash
[1] <s> [webpack.Progress] 91% sealing module assets processing
[1] <s> [webpack.Progress] 91% sealing module assets processing
[1] <s> [webpack.Progress] 91% sealing chunk assets processing
[1] <s> [webpack.Progress] 91% sealing chunk assets processing
[1] <s> [webpack.Progress] 92% sealing asset processing
[1] <s> [webpack.Progress] 92% sealing asset processing HotModuleReplacementPlugin
[1] <s> [webpack.Progress] 92% sealing asset processing PersistentChildCompilerSingletonPlugin
[1] <s> [webpack.Progress] 92% sealing asset processing SourceMapDevToolPlugin
[1] <s> [webpack.Progress] 92% sealing asset processing HtmlWebpackPlugin
[1] <s> [webpack.Progress] 92% sealing asset processing
[1] <s> [webpack.Progress] 93% sealing after asset optimization
[1] <s> [webpack.Progress] 93% sealing after asset optimization
[1] <s> [webpack.Progress] 93% sealing recording
[1] <s> [webpack.Progress] 93% sealing recording HotModuleReplacementPlugin
[1] <s> [webpack.Progress] 93% sealing recording
[1] <s> [webpack.Progress] 94% sealing after seal
[1] <s> [webpack.Progress] 94% sealing after seal
[1] <s> [webpack.Progress] 95% emitting emit
[1] <s> [webpack.Progress] 95% emitting emit DevMiddleware
[1] <s> [webpack.Progress] 95% emitting emit
[1] <s> [webpack.Progress] 98% emitting after emit
[1] <s> [webpack.Progress] 98% emitting after emit SizeLimitsPlugin
[1] <s> [webpack.Progress] 98% emitting after emit
[1] <s> [webpack.Progress] 99% done plugins
[1] <s> [webpack.Progress] 99% done plugins CaseSensitivePathsPlugin
[1] <s> [webpack.Progress] 99% done plugins webpack-dev-middleware
[1] <s> [webpack.Progress] 99% done plugins webpack-hot-middleware
[1] <s> [webpack.Progress] 99% done plugins
[1] <s> [webpack.Progress] 99% 
[1] 
[1] <s> [webpack.Progress] 99% cache store build dependencies
[1] <s> [webpack.Progress] 99% cache store build dependencies
[1] <s> [webpack.Progress] 99% cache begin idle
[1] <s> [webpack.Progress] 99% cache begin idle
[1] <s> [webpack.Progress] 100% 
[1] 
[1] ERROR in ./packages/components/src/button/index.tsx 5:12
[1] Module parse failed: Unexpected token (5:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  */
[1] | import clsx from 'clsx';
[1] > import type {
[1] |   ComponentPropsWithoutRef,
[1] |   ForwardedRef,
[1]  @ ./packages/components/src/button/stories/e2e/index.story.tsx 13:0-31 16:13-19 43:36-42 131:33-39 133:26-32 136:26-32 140:26-32 143:26-32
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./button/stories/e2e/index.story.tsx ./button/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/font-size-picker/index.tsx 4:12
[1] Module parse failed: Unexpected token (4:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  * External dependencies
[1] |  */
[1] > import type { ForwardedRef } from 'react';
[1] | 
[1] | /**
[1]  @ ./packages/components/src/font-size-picker/stories/e2e/index.story.tsx 13:0-35 17:13-27 28:27-41
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./font-size-picker/stories/e2e/index.story.tsx ./font-size-picker/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/h-stack/component.tsx 4:12
[1] Module parse failed: Unexpected token (4:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  * Internal dependencies
[1] |  */
[1] > import type { WordPressComponentProps } from '../context';
[1] | import { contextConnect } from '../context';
[1] | import { View } from '../view';
[1]  @ ./packages/components/src/h-stack/index.ts 1:0-48 1:0-48
[1]  @ ./packages/components/src/h-stack/stories/e2e/index.story.tsx 9:0-31 12:13-19 21:27-33
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./h-stack/stories/e2e/index.story.tsx ./h-stack/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/h-stack/hook.tsx 4:12
[1] Module parse failed: Unexpected token (4:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  * External dependencies
[1] |  */
[1] > import type { ReactElement } from 'react';
[1] | 
[1] | /**
[1]  @ ./packages/components/src/h-stack/index.ts 2:0-35 2:0-35
[1]  @ ./packages/components/src/h-stack/stories/e2e/index.story.tsx 9:0-31 12:13-19 21:27-33
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./h-stack/stories/e2e/index.story.tsx ./h-stack/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/popover/index.tsx 4:12
[1] Module parse failed: Unexpected token (4:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  * External dependencies
[1] |  */
[1] > import type { ForwardedRef, SyntheticEvent, RefCallback } from 'react';
[1] | import clsx from 'clsx';
[1] | import {
[1]  @ ./packages/components/src/popover/stories/e2e/index.story.tsx 9:0-28 13:13-20 23:65-72
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./popover/stories/e2e/index.story.tsx ./popover/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/v-stack/component.tsx 4:12
[1] Module parse failed: Unexpected token (4:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  * External dependencies
[1] |  */
[1] > import type { ForwardedRef } from 'react';
[1] | 
[1] | /**
[1]  @ ./packages/components/src/v-stack/index.ts 1:0-48 1:0-48
[1]  @ ./packages/components/src/v-stack/stories/e2e/index.story.tsx 9:0-31 12:13-19 21:27-33
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./v-stack/stories/e2e/index.story.tsx ./v-stack/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/v-stack/hook.ts 4:12
[1] Module parse failed: Unexpected token (4:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  * Internal dependencies
[1] |  */
[1] > import type { WordPressComponentProps } from '../context';
[1] | import { useContextSystem } from '../context';
[1] | import { useHStack } from '../h-stack';
[1]  @ ./packages/components/src/v-stack/index.ts 2:0-35 2:0-35
[1]  @ ./packages/components/src/v-stack/stories/e2e/index.story.tsx 9:0-31 12:13-19 21:27-33
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./v-stack/stories/e2e/index.story.tsx ./v-stack/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/view/component.tsx 14:12
[1] Module parse failed: Unexpected token (14:12)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |  * Internal dependencies
[1] |  */
[1] > import type { WordPressComponentProps } from '../context';
[1] | 
[1] | const PolymorphicDiv = styled.div``;
[1]  @ ./packages/components/src/view/index.ts 1:0-46 1:0-46
[1]  @ ./packages/components/src/v-stack/stories/e2e/index.story.tsx 8:0-37 27:84-88
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./v-stack/stories/e2e/index.story.tsx ./v-stack/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./storybook/preview.js 117:3
[1] Module parse failed: Unexpected token (117:3)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] |           // so the component is always visible before the fold.
[1] |           page: () => (
[1] >                   <>
[1] |                           <Title />
[1] |                           <Subtitle />
[1]  @ ./test/storybook-playwright/storybook/preview.js 5:0-64 9:4-33 25:4-32 28:31-59
[1]  @ ./storybook-config-entry.js 8:150-241 29:2-32:4 29:207-32:3
[1] 
[1] ERROR in ./packages/components/src/button/stories/e2e/index.story.tsx 8:0-45
[1] Module not found: Error: Can't resolve '@wordpress/icons' in '/home/username/projects/gutenberg/packages/components/src/button/stories/e2e'
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./button/stories/e2e/index.story.tsx ./button/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/font-size-picker/stories/e2e/index.story.tsx 8:0-46
[1] Module not found: Error: Can't resolve '@wordpress/element' in '/home/username/projects/gutenberg/packages/components/src/font-size-picker/stories/e2e'
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./font-size-picker/stories/e2e/index.story.tsx ./font-size-picker/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./packages/components/src/popover/stories/e2e/index.story.tsx 4:0-46
[1] Module not found: Error: Can't resolve '@wordpress/element' in '/home/username/projects/gutenberg/packages/components/src/popover/stories/e2e'
[1]  @ ./packages/components/src/ lazy ^\.\/.*$ include: (?%21.*node_modules)(?:\/packages\/components\/src(?:\/(?%21\.)(?:(?:(?%21(?:^%7C\/)\.).)*?)\/%7C\/%7C$)stories\/e2e\/(?%21\.)(?=.)[^/]*?\.story\.(js%7Ctsx%7Cmdx))$ chunkName: [request] namespace object ./popover/stories/e2e/index.story.tsx ./popover/stories/e2e/index.story
[1]  @ ./storybook-stories.js 10:11-14:5
[1]  @ ./storybook-config-entry.js 6:0-50 17:31-39 24:2-27:4 24:58-27:3 26:31-39
[1] 
[1] ERROR in ./test/storybook-playwright/storybook/decorators/with-custom-controls.js 11:9
[1] Module parse failed: Unexpected token (11:9)
[1] You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
[1] | 
[1] |   if ( context.globals.customE2EControls === 'hide' ) {
[1] >           return <Story { ...context } />;
[1] |   }
[1] | 
[1]  @ ./test/storybook-playwright/storybook/preview.js 6:0-71 26:1-19
[1]  @ ./storybook-config-entry.js 8:150-241 29:2-32:4 29:207-32:3
[1] 
[1] preview compiled with 13 errors
[1] => Failed to build the preview
[1] <s> [webpack.Progress] 99% end closing watch compilation
[1] <s> [webpack.Progress] 99% end closing watch compilation
[1] WARN Force closed preview build
[1] SB_BUILDER-WEBPACK5_0003 (WebpackCompilationError): There were problems when compiling your code with Webpack.
[1] Run Storybook with --debug-webpack for more information.
[1]     at starter (./node_modules/@storybook/builder-webpack5/dist/index.js:1:23903)
[1]     at starter.next (<anonymous>)
[1]     at Module.start (./node_modules/@storybook/builder-webpack5/dist/index.js:1:25882)
[1]     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[1] 
[1] WARN
[1]  Broken build, fix the error above.
[1] WARN You may need to refresh the browser.
```

</details> 

### Step-by-step reproduction instructions

```
npm cache clean --force
rm -rf ~/.npm
rm -rf node_modules/
npm install
npm run clean:package-types
npm run clean:packages
npm run storybook:e2e:dev
```

### Environment info

- Gutenberg version: trunk (61d7404cea)
- OS: Windows WSL2 (Ubuntu 22.04.4 LTS)

--- a/test/storybook-playwright/storybook/main.js
+++ b/test/storybook-playwright/storybook/main.js
@@ -5,7 +5,10 @@ const baseConfig = require( '../../../storybook/main' );
 
 const config = {
 	...baseConfig,
-	addons: [ '@storybook/addon-toolbars' ],
+	addons: [
+		'@storybook/addon-toolbars',
+		'@storybook/addon-webpack5-compiler-babel',
+	],
 	docs: undefined,
 	staticDirs: undefined,
 	stories: [

